icu4c/source/configure.ac

```diff
@@ -1462,4 +1464,11 @@ then
         echo "## Expect build failures in the 'data', 'test', and other directories."
 fi
 
+if test -z "$PYTHON3";
+then
+    echo "##"
+    echo "## Notice: Python 3 not found. ICU 64 will require Python 3 to build. See ICU-10923."
```

Comment:
suggestion: 
- `ICU versions 64 and later will require…`
- make the ticket number have a URL (and include a link to Python on the ticket)